packages/wp-now/src/tests/wp-now.spec.ts

```diff
@@ -506,3 +509,34 @@ describe('Test starting different modes', () => {
 		expect(themeName.text).toContain('Twenty Twenty-Three');
 	});
 });
+
+/**
+ * Test wp-cli command.
+ */
+describe('wp-cli command', () => {
+	beforeAll(async () => {
+		fs.rmSync(getWpNowTmpPath(), { recursive: true, force: true });
+		console.time('wp-cli');
+		await downloadWPCLI();
+		console.log('wp-cli downloaded.');
+		console.timeEnd('wp-cli');
+	});
+
+	afterAll(() => {
+		fs.removeSync(getWpCliPath());
+	});
+	/**
+	 * Test wp-cli displays the version.
+	 * We don't need the WordPress context for this test.
+	 */
+	test('wp-cli displays the version', async () => {
+		let output = '';
+		function collectOutput(outputLine: string) {
+			output += outputLine;
+		}
+		await executeWPCli(['--version'], {
```

Comment:
```suggestion
		await executeWPCli(['cli info'], {
```

Reply:
Whomp. Should we include some note in the code about how it's an unstable API?

Reply:
Added a comment here:
https://github.com/WordPress/wordpress-playground/pull/395/commits/e0395e61c9135f59d7d7af1859dae178b488ea67